wger/core/tests/test_create_user_via_api.py

```diff
@@ -0,0 +1,58 @@
+# This file is part of wger Workout Manager.
+#
+# wger Workout Manager is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# wger Workout Manager is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+import json
+
+from rest_framework import status
+
+from django.contrib.auth.models import User
+from wger.core.models import UserProfile
+
+from wger.core.tests.base_testcase import WorkoutManagerTestCase
+
+
+class ApiUserTestCase(WorkoutManagerTestCase):
+    '''
+     Test API user creation
+    '''
+
+    def test_create_user_via_api(self):
+        '''
+        Tests the creation of a user via the API
+        '''
+        user = User.objects.create_user(
+            username='FitnessFirst',
+            email='email@oduk.com',
+            password='password')
+        user.save()
+        profile = UserProfile.objects.get(user=user)
+        profile.can_create_via_api = True
```

Comment:
I think there is a way to generate an API Key for the user which you can pass in the Headers. I guess that would be a better implementation.